packages/cli/src/utils/systems/registerFunctionCalls.ts

```diff
@@ -0,0 +1,61 @@
+import { tableIdToHex } from "@latticexyz/common";
+import { System } from "./types";
+import { loadFunctionSignatures, toFunctionSelector } from "./utils";
+import { CallData } from "../utils/types";
+
+export function registerFunctionCalls(input: {
```

Comment:
should we call this `getRegisterFunctionSelector` to make it consistent with the other function names? Or could even rename all to `get<MethodName>CallData` to make it extra clear (i was surprised for a second that this function doesn't actually `register` yet but just prepares the call by returning the calldata)